mobx_codegen/test/data/valid_input_to_string_true.dart

```diff
@@ -0,0 +1,84 @@
+library generator_sample;
+
+import 'package:meta/meta.dart';
+import 'package:mobx/mobx.dart';
+//import 'package:flutter/material.dart';
+
+part 'generator_sample.g.dart';
+
+typedef VoidCallback = void Function();
+
+class User = UserBase with _$User;
+
+@MakeStore(generateToString: true)
```

Comment:
We don't have store annotations anymore

Reply:
> I will change all of this. Since we won't have store annotation anymore should I create a @tostring annotation ?

I think its still worth having a toString. Users can always override if they want to specialize differently.

Reply:
Except they'd have to override in the public class, which most people write without a body.

As a user, I'd prefer to have this flagged. Having it on by default will add a lot of work for me.

Reply:
The other option is we introduce a `@codegen({toString: true})` style of annotation, which can be extended in the future ?